src/libraries/System.Net.WebSockets.Client/tests/ClientWebSocketTestBase.cs

```diff
@@ -26,6 +26,7 @@ public class ClientWebSocketTestBase
         }).ToArray();
 
         public const int TimeOutMilliseconds = 20000;
+        public const int BrowserTimeOutMilliseconds = 30000;
```

Comment:
as far as I can tell, this is just upper boundary. I think it would be ok to just bump up TimeOutMilliseconds instead of creating new variable. If you feel this should be different, I would put the logic here instead of dragging Browser branches through the tests.

Reply:
Addressed in recent commits